src/content/reference/rules/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Rules of React
+---
+
+<Intro>
+Just as different programming languages have their own ways of expressing concepts, React has its own idioms — or rules — for how to express patterns in a way that is easy to understand and yields high-quality applications. This page lists all the Rules of React.
+</Intro>
+
+<InlineToc />
+
+---
+
+<Note>
+To learn more about expressing UIs with React, we recommend reading [Thinking in React](/learn/thinking-in-react). This section describes the React programming paradigm from the perspective of its key rules, which may differ from approaches you've used elsewhere. Many of these rules are common to other UI libraries and non-UI related "reactive" systems, in addition to being a strong approach to building robust software generally.
```

Comment:
> This section describes the React programming paradigm from the perspective of its key rules, which may differ from approaches you've used elsewhere. Many of these rules are common to other UI libraries and non-UI related "reactive" systems, in addition to being a strong approach to building robust software generally.

I think taking another approach to these sentences would help readers that aren't React experts understand what this section is about. I'd rewrite this to say something more along these lines:


This section describes the rules you need to follow to write idiomatic React code. Writing idiomatic React code can help you write well organized, safe, and composable applications. These properties make your app more resilient to changes and makes it easier to work with other developers, libraries, and tools.

Reply:
Love it, incorporated!